components/menu/test/menu.test.js

```diff
@@ -55,13 +91,16 @@ describe('d2l-menu', () => {
 			elem = await fixture(html`
 				<d2l-menu>
 					<d2l-menu-item id="hidden_a" text="a" hidden></d2l-menu-item>
-					<d2l-menu-item id="a1" text="a"></d2l-menu-item>
+					${delayedMenuItem}
 					<d2l-menu-item id="b1" text="b" disabled></d2l-menu-item>
 					<d2l-menu-item id="a2" text="a"></d2l-menu-item>
 					<d2l-menu-item id="c1" text="C"></d2l-menu-item>
 					<d2l-menu-item id="d1" text="d"></d2l-menu-item>
 				</d2l-menu>
 			`);
+			const delayed = elem.querySelector('#a1');
+			await delayed.updateComplete;
+			await nextFrame();
 			await nextFrame();
```

Comment:
This is not great, but seems stable locally.  We'll see what CI thinks.  One `nextFrame` makes the tests flaky - maybe a red flag about this change or about how the localize mixin works in general